rust/lsp-lib/src/utils.rs

```diff
@@ -205,4 +205,11 @@ pub fn start_new_server(
     }
 
     Ok(language_server_client)
+}
+
+/// Helper Method to get Url of the View path
+/// Panics: If view does not have a path or the given path
+/// can not be converted to Url
+pub fn get_view_uri<C: Cache>(view: &View<C>) -> Option<Url> {
```

Comment:
If this is the only place that you're using the `bail` macro, and you want to have some fun, you could consider just writing a `uri_for_view!` macro?